gammapy/background/tests/test_reflected.py

```diff
@@ -62,29 +63,39 @@ def test_find_reflected_regions(exclusion_mask, on_region):
     )
     fregions.run()
     regions = fregions.reflected_regions
-    assert len(regions) == 14
-    assert_quantity_allclose(regions[3].center.icrs.ra, Angle("83.674 deg"), rtol=1e-2)
+    assert len(regions) == 15
```

Comment:
Where does the change come from? Bin size?

Reply:
My guess is that it comes from the fact that the logics to determine the starting angle of OFF regions has been changed.